mobile/src/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pressable, Text, View } from "react-native";
+/**
+ * name - name of the button.
+ * addButtonStyle - add custom style to the button.
+ * addTextStyle - add custom style to the text.
+ * pilled - make the button pilled.
+ **/
+export default function Buttons({
```

Comment:
Should be `Button`, no?

Reply:
I was hesitant to do that because Button is reserved already but it should be fine as long as we don't import ```Button``` from react native.